### ISSUE TITLE
Move reading pop data to main module - fill continents pages

### ISSUE THREAD

**Issue by GuidoMaggio** [miraisolutions/Covid19#92]
get_pop_data is called in multiple places. It should be done only once in app_server

**Comment by GuidoMaggio** [comment]
Reading now pop data only in app_server
Had to change few things in other modules to make it possible
changed many example
Added datasets to construct popUN.csv which now contains also continents and subcontinents
A page comparing continents could now be constructed

**Comment by GuidoMaggio** [comment]
Merged issue 94 with continents page

**Comment by GuidoMaggio** [comment]
![image](https://user-images.githubusercontent.com/13679154/84486643-fba5a000-ac9d-11ea-959a-fe422a10a42b.png)
Added maps of countries within continents in continens tabs

**Comment by GuidoMaggio** [comment]
Maps
Adjusted the zoom to center always the mean of latitude and longitude coordinates. In this way when browser changes size then the continent is always centered correctly.
Added keepInView = T, autoPan = F
dragging = true to allow viewing popup falling outside map 

Replaced population data with UN data.
I realized the the population size was wrong  in the current dataset for some small countries like Iceland: 36426. This creates a problem in the map because such countries result having too high cases over 1M.
Pop size now are slightly different for main countries, being them an UN estimate for 2020 Q2
